mockintosh/handlers.py

```diff
@@ -259,6 +273,7 @@ def initialize(
             self.unhandled_data = unhandled_data
             self.fallback_to = fallback_to
             self.tag = tag
+            self.remove_content_type_header = True
```

Comment:
If you overridden `clear()` to not set content-type header, why do we need this flag?

Reply:
@undera even though I've overridden the `clear()` I saw `Content-Type=text/html` header while building the replica response object in [here](https://github.com/up9inc/mockintosh/blob/fix/tornado-content-type-header/mockintosh/handlers.py#L564-L565). That's why that `remove_content_type_header` attribute exists. Though I'm unable to reproduce what I did to get `Content-Type=text/html` header there yesterday. Let me try a few more configurations and if I cannot get `Content-Type=text/html` header then I'll remove the `remove_content_type_header` attribute.

Reply:
I've removed the `remove_content_type_header` attribute.